code/modules/economy/holopay.dm

```diff
@@ -0,0 +1,249 @@
+/// Max range at which the hologram can be projected before it deletes
+#define MAX_HOLO_RANGE 4
```

Comment:
If you only use a value once and its purpose is clear, you don't need to define it. 
Defines/the magic number rule are built to bully people from using asspull constants that hold meaning that's not clear from just their usage.